TRIAGE.md

```diff
@@ -0,0 +1,65 @@
+| `enhancement`            | Indicates that the issue suggests an improvement or new feature.          |
+| `documentation`          | Indicates improvements or additions to documentation.                     |
+| `roadmap`                | Indicates epics and tasks related to roadmap items.                       |
+| `programs`               | Indicates epics and tasks related to Community programs.                  |
+| `team`                   | Indicates issues for team collaboration.                                  |
+| `adr-required`           | Indicates issues requiring Architectural Decision Record                  |
+
+Severity labels:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `High priority`          | High priority flow or feature is broken/non-functional, crash.            |
+| `Critical priority`      | Indicates Critical impact on data or site: data loss (posts, pages, comments), unexpected publishing, security issue.|
+
+
+
+Other labels:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `good first issue`       | Indicates that the issue is suitable for newcomers to contribute to the project.|
+| `question`               | Indicates that the issue lacks sufficient information to be able to act on it right now and requires additional details from the reporter. The maintainer for the area must remove the tag when more info is provided, or close it if it is too old.|
+| `duplicate`              | Indicates that is duplicated.                                             |
+
+## Triage process
+
+At the end of reading an issue, you should do one of the following:
+
+1. Make sure to search in previous issue to ensure this is not a duplicated issue. If that is the case, mark it as `duplicate` and close it referencing the original issue. If the issue is clear, ensure it has the correct `area` assigned. Also consider if it is a `good first issue`.  
+
+2. Asses the issue severity to determine what functionality is affected and how broken it is and use the severity labels just for High and Critical cases.
+
+3. If the submitter or the reviewer believe the resolution of the Issue is likely to result in a significant or noteworthy change it will be labeled as `adr-required`.
+
+4. Move the issue to the correct repo if required.
+
+5. If it's not clear what the problem or issue is, add a note for the requestor (or occasionally some other user on the thread), and mark it as `question`. If a `question` issue persists for longer than a week or so (past a second followup), it's reasonable to `close` the issue and encourage the requester to reopen if they have more details.
```

Comment:
I think this needs revision as per my other comments.

Reply:
This is fixed in the new version of the PR.